lib/src/DynamicProto.ts

```
@@ -285,8 +302,14 @@ function _getBaseFuncs(classProto:any, thisTarget:any, instFuncs:any, useBaseIns
         }
 
         return function() {
-            // eslint-disable-next-line prefer-rest-params
-            return theFunc.apply(target, arguments);
+            try {
```

Revert and let this throw any exception back to the caller.